buyer-api/src/queues/notarizationQueue.js

```diff
@@ -1,11 +1,96 @@
+import uuid from 'uuid/v4';
 import { createQueue } from './createQueue';
-import { logger } from '../utils';
+import {
+  dataResponses,
+  dataResponsesBatches as batches,
+  notarizations,
+} from '../utils/stores';
+import logger from '../utils/logger';
+import config from '../../config';
+
+const { buyerPublicBaseUrl } = config;
+
+/**
+ * @async
+ * @function createNotarizationRequest
+ *  Creates a Notarization record containing the NotarizationRequest.
+ * @param {string} notaryAddress Notary's Ethereum address
+ * @param {number} orderId Order ID in the DataExchange contract
+ * @param {import('../utils/stores').NotarizationSeller[]} sellers List of sellers data
```

Comment:
We can use an alias for this typedef on this file, that way `@param` tag looks cleaner.